src/multiLanguageDocumentList.js

```diff
@@ -0,0 +1,249 @@
+import S from '@sanity/desk-tool/structure-builder'
+import { liveDocuments } from '@kaliber/sanity-live-documents'
+import { map } from 'rxjs/operators'
+import React from 'react'
+import * as rxjs from 'rxjs'
+import { Title } from './Title'
+import schema from 'part:@sanity/base/schema'
+import { usePane, usePaneLayout } from '@sanity/desk-tool/lib/components'
+import { usePaneRouter } from '@sanity/desk-tool'
+import { ListPaneContent } from './machinery/sanity/ListPaneContent'
+import { PaneItem } from './machinery/sanity/PaneItem'
+import { useDeskToolSettings } from './machinery/sanity/useDeskToolSettings'
+import * as uuid from 'uuid'
+
+export function multiLanguageDocumentList(listBuilder) {
+  const serializedListBuilder = listBuilder.serialize()
+
+  const emptyDoc = { _type: serializedListBuilder.schemaTypeName }
+  const document$ = new rxjs.BehaviorSubject(emptyDoc)
+  const component = React.memo(React.forwardRef(MultiLanguageDocumentList))
+
+  return {
+    ...serializedListBuilder,
+    type: 'component', component,
+    key: serializedListBuilder.id, // override the key to prevent an unmount / mount cycle
+    document$,
+    child: translationId => document$.pipe(
+      map(_doc => {
+        const docValid = _doc.translationId && _doc.translationId === translationId
+        const doc = docValid ? _doc : emptyDoc
+        return getDocumentNode({ schemaType: doc._type, documentId: doc._id || uuid.v4() })
+          .id(doc.translationId || translationId)
+          .initialValueTemplate(serializedListBuilder.schemaTypeName, { translationId })
+      })
+    )
+    // Ik denk dat we in de actiebalk vlaggetjes moeten plaatsen waarmee je kunt switchen voor de display van de taal (tenzij een document niet beschikbaar is in die taal)
```

Comment:
We zouden dat wellicht ook voor niet beschikbare talen kunnen doen, als we die op de een of andere manier duidelijk kunnen onderscheiden. De opties (copy of blanco) kunnen we in een [MenuButton](https://www.sanity.io/ui/docs/component/menu-button) aanbieden